osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
I think the sensed contour should be given directly in sensor coordinates. This is more natural for a sensor, because a sensor does not sense a position in the middle of a bounding box or where ever, but it senses the contour of an object in sensor coordinates. 
I agree with @AndreasVIRES that we should discuss the coordinate topic in general all together during the workshop.